sentiment-analysis/main.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/http;
+
+configurable string googleapisAPIKey = ?;
+
+type Document record {|
+    string 'type = "PLAIN_TEXT";
+    string content;
+|};
+
+type AnalyseRequest record {|
+    string encodingType = "UTF8";
+    Document document;
+|};
+
+type SentimentResponse record {
+    DocumentSentiment documentSentiment;
+};
+
+type DocumentSentiment record {|
+    float magnitude;
+    float score;
+|};
+
+final http:Client sentimentClient = check new ("https://language.googleapis.com");
```

Comment:
This sample is then again another HTTP EP integration; my suggestion was to use openAPI client to do the sentiment analysis. Please refer https://ballerina.io/usecases/ai/

Reply:
updated.